packages/react-components/src/components/VideoTile.tsx

```diff
@@ -151,7 +163,47 @@ const DefaultPlaceholder = (props: CustomAvatarOptions): JSX.Element => {
   );
 };
 
+/* @conditional-compile-remove(pinned-participants) */
+/**
+ * @internal
+ */
+export type VideoTileMenuItem = {
+  key: string;
+  ariaLabel?: string;
+  text: string;
+  onClick: () => void;
+  icon: IIconProps;
+};
+
+/* @conditional-compile-remove(pinned-participants) */
+/**
+ * @internal
```

Comment:
Should need to be changed to @beta as this should be the public facing type

Reply:
done